doc/Caliptra.md

```diff
@@ -935,6 +937,19 @@ To anticipate this eventuality, before a power cycle the SoC can instruct DPE to
 
 Note: as all DPE leaf keys are derived using Caliptra runtime firmware's CDI, a DPE simulation context cannot predict the leaf key that would be available if a different Caliptra firmware image were to boot (as one Caliptra firmware image does not have access to a different image's CDI). Therefore, if a different Caliptra firmware image is staged to persistent storage, Caliptra must first be hitlessly updated to that image before a simulation context can be used to predict public keys that will be available to that image on next cold-boot.
 
+### Attestation of SoC update journey
+
+Caliptra shall also attest to the journeys of SoC components. A SoC component's journey may change independently of other components. For example, SoC components may implement partial resets or hitless updates that cause the component's firmware or configuration to reload.
+
+The Caliptra DPE shall maintain a reboot counter for each component. Caliptra DPE shall increment the reboot counter and update the journey measurement for DPE calls that indicate that the component's state changed. Caliptra DPE shall attest both the counter and the journey measurement. The verifier is assumed to have knowledge of update events at an associated reboot counter (via an event log) but not have knowledge of reset events. The verifier can compute the journey measurement via multiplicatively extending the current measurement by the reset counter. For example:
```

Comment:
"Caliptra shall attest both the counter and the journey measurement"

It seems to me that the counter need only be informationally reported by Caliptra. If the counter is ever modified, the journey verification will fail.

Reply:
Right it doesn't have to be cryptographically attested to, but needs to be included as part of attestation collateral for the verification to pass. Do you have a rephrase in mind?

Reply:
Do we have the accompanying evidence documented. I recall you had a document, but do we want to make that part of the spec.

Reply:
First stab at suggested reword: "Caliptra DPE shall attest to the journey measurement, and report the counter on-demand."